payments/models.py

```diff
@@ -0,0 +1,83 @@
+
+import uuid
+from django.core.urlresolvers import reverse
+from django.db import models
+import getpaid
+
+
+CURRENCIES = (
+    ('DKK', 'DKK'),
+    ('PLN', 'PLN'),
+    ('GBP', 'GBP'),
+)
+
+
+class OrderManager(models.Manager):
+
+    def create_for_fee(self, user, fee):
+        name = "Order for {}".format(fee.name)
+        instance = self.model(name=name,
+                              total=fee.amount,
+                              user=user,
+                              currency=fee.currency)
+        instance.save()
+        return instance
+
+
+class Order(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    name = models.CharField(max_length=100)
+    total = models.DecimalField(decimal_places=2, max_digits=8, default=0)
+    currency = models.CharField(max_length=3, default='DKK',
+                                choices=CURRENCIES)
+    user = models.ForeignKey('auth.User')
```

Comment:
I remember that @valberg didn't want to have custom user models so I put it as django's default - this will expand to `django.contrib.auth.models.User`; but I can change it to `get_user_mode()l` call if this is what you want?